EIPS/eip-4519.md

```diff
@@ -0,0 +1,237 @@
+---
+eip: 4519
+title: Smart Non Fungible Token (SmartNFT)
+description: A standard interface of smart non-fungible tokens representing smart assets (such as secure IoT devices) that can generate their own blockchain accounts and obey users.
+author: Javier Arcenegui <arcenegui@imse-cnm.csic.es>, Rosario Arjona <arjona@imse-cnm.csic.es>, Roberto Román <roman@imse-cnm.csic.es> and Iluminada Baturone <lumi@imse-cnm.csic.es>
```

Comment:
One of the authors _must_ have a github username associated with the EIP so that the EIP bot will function correctly.